pages/userSettings/userSettings.ejs

```diff
@@ -13,6 +13,59 @@
     <div id="content" class="container">
       <h1 class="mb-4">Settings</h1>
 
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex align-items-center">
+          Logged-in user profile information
+          <a href="/pl/logout" class="btn btn-danger btn-sm ml-auto">Logout</a>
+        </div>
+          <table class="table table-sm two-column-description">
+            <tbody>
+              <tr>
+                <th>User ID (UID)</th>
+                <td>
+                  <%= authn_user.uid %>
+                </td>
+              </tr>
+              <tr>
+                <th>User Name</th>
+                <td><%= authn_user.name %></td>
+              </tr>
+              <tr>
+                <th>Unique Id Number (UIN)</th>
+                <td><%= authn_user.uin %></td>
+              </tr>
+              <tr>
+                <th>Institution</th>
+                <td><%= authn_institution.long_name %> (<%= authn_institution.short_name %>)</td>
+              </tr>
+              <tr>
+                <th>Authentication method</th>
+                <td>
+                  <%= authn_provider_name %>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+      </div>
+
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex">
+          Browser configuration
+        </div>
+        <div class="card-body">
+          <p>These options will let you reset some browser settings related to technology inside PrairieLearn.
+            Use these buttons to reset things if things are broken for you.
+          </p>
+          <p>
+          <button class="btn btn-md btn-info"
+            onClick="localStorage.removeItem('MathJax-Menu-Settings');alert('MathJax menu settings have been reset');">
+            Reset MathJax menu settings
```

Comment:
```suggestion
            Reset MathJax settings
```

Reply:
I'm going to leave this one as is, as the menu name hints where the settings got set from. There could be other reset buttons in the future.